fedora/Dockerfile

```diff
@@ -1,13 +1,9 @@
-FROM fedora:20
+FROM fedora
 MAINTAINER amitsaha.in@gmail.com
 
-# Let's start with some basic stuff.
-RUN yum -y clean all
-RUN yum -y update
-RUN yum install -y iptables ca-certificates lxc e2fsprogs
-
-# Install Docker from Fedora repos
-RUN yum -y install docker-io
+# Update and install docker
+RUN dnf -y update && dnf clean all
+RUN dnf -y install docker && dnf clean all
```

Comment:
Well, usually, those lines are cached, so them taking longer shouldn't matter much. Combining them, makes the install an "atomic" action, and is considered best practice. i.e. "I want to update _and_ install from those updated files".

If you leave them separate statements, Changing the "install" line, will only invalidate the cache for that line, and *not* run the "update" again, possibly leading to installing outdated packages.

Really not a big issue, just a "nit".

Also, I'm not a maintainer here, only commenting, so best left to @jpetazzo to decide :+1: